bin/learnyounode

```diff
@@ -1,3 +1,7 @@
 #!/usr/bin/env node
 
-require('../learnyounode').execute(process.argv.slice(2))
\ No newline at end of file
+require('../learnyounode').execute(process.argv.slice(2))
+
+if(process.argv.slice(2) == 0){
```

Comment:
_(This is not standard syntax)_

Workshopper has a [`commands` option](https://github.com/workshopper/workshopper-adventure/blob/7c0493edc6b4374be60898c08fa23130e65aa7d7/index.js#L102) that allows to register custom commands just for this purpose. Of course there should be a [`postinstall`](https://docs.npmjs.com/misc/scripts) script that launches this command.

Reply:
Huge thanks Martin. I'll get these knocked out this week.